lib/AST/Util.cpp

```diff
@@ -316,6 +316,26 @@ clang::Expr *Clone(clang::ASTUnit &unit, clang::Expr *expr,
   return CHECK_NOTNULL(cloner.Visit(CHECK_NOTNULL(expr)));
 }
 
+static clang::Expr *StripParens(clang::Expr *expr) {
```

Comment:
Can this be replaced by `clang::Expr::IgnoreParens()` ?

Reply:
Done!